_includes/pbc/all/install-features/202311.0/install-glue-api/install-the-shipment-glue-api.md

```diff
@@ -0,0 +1,488 @@
+
+
+
+This document describes how to install the [Shipment](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/shipment-feature-overview.html) API.
+
+## Install feature core
```

Comment:
Since there is only the feature core to install, do we need to mention it?

Reply:
IDK, it's a part of our integration guide template, if you think it's redundant we can remove it